curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -532,6 +533,9 @@ MdsClientImpl::RefreshSession(const std::vector<PartitionTxId> &txIds,
             LOG(INFO) << "RefreshSession need update partition txid list: "
                       << response.DebugString();
         }
+        if (*enableSumInDir == true && response.enablesumindir() == false) {
+            *enableSumInDir = response.enablesumindir();
```

Comment:
use enableSumInDir->store(response.enablesumindir());

And change use enableSumInDir_ value to enableSumInDir_.load()

std::atomic::store and std::atomic::load is thread safety.

Reply:
Get it! I will change afternoon.